.github/workflows/core.yml

```diff
@@ -71,6 +71,8 @@ jobs:
         with:
           activate-environment: ${{ env.IDAES_CONDA_ENV_NAME_DEV }}
           python-version: ${{ matrix.python-version }}
+      - name: Install additional dependencies
+        run: pip install idaes-pse
```

Comment:
This would install the latest _released_ version of `idaes-pse`. I'd suggest:

- Create a file named `requirements-dev.txt` with the following content:
  ```txt
  --editable .[dev]
  idaes-pse @ git+https://github.com/IDAES/idaes-pse@main
  ```
- Remove this step
- Replace `install-target: .[dev]` with `install-target: -r requirements-dev.txt` in the step below
- (If needed) update the developer installation instructions with `pip install -r requirements-dev.txt`

Reply:
Is the benefit of making this change that we can edit the `requirements-dev.txt` file to alter the `idaes-pse version`, rather than editing the `core.yaml`?

Reply:
It's intended to make the interface to users simpler, and more similar to idaes-pse